package.json

```diff
@@ -57,7 +57,7 @@
         "eslint-plugin-license-header": "0.6.1",
         "jsdom": "25.0.1",
         "set-value": "4.1.0",
-        "typescript": " 5.5.4",
+        "typescript": "^5.6.3",
```

Comment:
I am using only the exact version in the `package.json` file. Can you change that?

Reply:
Right, of course!